packages/ckeditor5-dev-env/lib/utils/changelog.js

```diff
@@ -0,0 +1,249 @@
+	 */
+	changelogHeader: `Changelog\n=========\n\n`,
+
+	/**
+	 * Returns a type (major, minor, patch) of the next release based on commits.
+	 *
+	 * @returns {Promise}
+	 */
+	getNewReleaseType() {
+		const conventionalRecommendedBump = require( 'conventional-recommended-bump' );
+		const parserOpts = require( '../changelog/parser-opts' );
+
+		return new Promise( ( resolve, reject ) => {
+			const options = {
+				whatBump: utils.getNewVersionType
+			};
+
+			conventionalRecommendedBump( options, parserOpts, ( err, response ) => {
+				if ( err ) {
+					return reject( err );
+				}
+
+				return resolve( response );
+			} );
+		} );
+	},
+
+	/**
+	 * Returns version of current package.
+	 *
+	 * @param {String} [cwd=process.cwd()] Current work directory.
+	 * @returns {String}
+	 */
+	getCurrentVersion( cwd = process.cwd() ) {
+		const packageJSON = require( path.resolve( cwd, 'package.json' ) );
+
+		return `v${ packageJSON.version }`;
+	},
+
+	/**
+	 * Returns number of the next version.
+	 *
+	 * @param {String} currentVersion Current version in format "X.Y.Z".
+	 * @param {String} releaseType Type of the next release.
+	 * @returns {String}
+	 */
+	getNextVersion( currentVersion, releaseType ) {
+		if ( currentVersion.startsWith( 'v' ) ) {
+			currentVersion = currentVersion.slice( 1 );
+		}
+
+		const version = currentVersion.split( '.' ).map( ( n ) => parseInt( n ) );
+
+		if ( releaseType === 'major' ) {
+			return `${ version[ 0 ] + 1 }.0.0`;
+		} else if ( releaseType === 'minor' ) {
+			return `${ version[ 0 ] }.${ version[ 1 ] + 1 }.0`;
+		}
+
+		return `${ version[ 0 ] }.${ version[ 1 ] }.${ version[ 2 ] + 1 }`;
+	},
+
+	/**
+	 * Returns a level which represents a type of release based on the commits.
+	 *   - 0: major,
+	 *   - 1: minor,
+	 *   - 2: patch.
+	 *
+	 * An input array is a result of module {@link https://github.com/conventional-changelog/conventional-commits-parser}.
+	 *
+	 * @param {Array} commits
+	 * @returns {Number}
+	 */
+	getNewVersionType( commits ) {
+		let hasNewFeatures = false;
+
+		for ( const item of commits ) {
+			for ( const note of item.notes ) {
+				if ( note.title === 'BREAKING CHANGE' ) {
+					return 0;
+				}
+			}
+
+			if ( !hasNewFeatures && item.type === 'Feature' ) {
+				hasNewFeatures = true;
+			}
+		}
+
+		return hasNewFeatures ? 1 : 2;
+	},
+
+	/**
+	 * Parses command line arguments and returns them as a user-friendly hash.
+	 *
+	 * @returns {Object} options
+	 * @returns {String} options.token GitHub token used to authenticate.
+	 * @returns {Boolean} options.init Whether to create first release using this package.
+	 * @returns {Boolean} options.debug Whether to show additional logs.
+	 */
+	parseArguments( args = process.argv.slice( 2 ) ) {
+		const options = require( 'minimist' )( args, {
+			string: [
+				'token'
+			],
+
+			boolean: [
+				'init',
+				'debug'
+			],
+
+			default: {
+				init: false,
+				debug: false
+			}
+		} );
+
+		delete options._;
```

Comment:
Please remove this or comment it and write a test for it.

Hint: Since it'd be a waste of time to test it it means that this code makes no sense and could be removed :D